react-native-libraries.json

```
@@ -15275,5 +15275,17 @@
     "expoGo": true,
     "newArchitecture": true,
     "fireos": true
-  }
+  },
+  {
+  "githubUrl": "https://github.com/iosazee/expo-passkey",
+  "npmPkg": "expo-passkey",
+  "examples": [
+    "https://github.com/iosazee/neb-starter"
+  ],
+  "ios": true,
+  "android": true,
+  "web": false,
+  "expoGo": false,
+  "newArchitecture": true
+}
```

```suggestion
    "githubUrl": "https://github.com/iosazee/expo-passkey",
    "examples": [
      "https://github.com/iosazee/neb-starter/tree/main/apps/mobile"
    ],
    "ios": true,
    "android": true,
    "newArchitecture": true
  }
```

Data cleanup, example correction, whitespace nit.